.github/workflows/ci.yml

```diff
@@ -173,6 +167,14 @@ jobs:
         export PATH=$PATH:${GITHUB_WORKSPACE}/install/bin:${VCPKG_INSTALLATION_ROOT}/installed/x64-windows/bin
         cmake --build . --config ${{ matrix.build_type }}
 
+    - name: Test
+      shell: bash
+      run: |
+        cd build
+        cmake -DBUILD_TESTING:BOOL=ON .
+        cmake --build . --config ${{ matrix.build_type }}
```

Comment:
I do not think you need to rebuild as `BUILD_TESTING` was already enabled during the build, but for sure it does not harm either.

Reply:
Yes, but better off cleaning it up.